app/src/main/java/org/wikipedia/ActivityLifecycleHandler.java

```diff
@@ -79,4 +81,19 @@ public void onActivityDestroyed(@NonNull Activity activity) {
             haveMainActivity = false;
         }
     }
+
+    @Override
+    public void onConfigurationChanged(@NonNull Configuration configuration) {
+    }
+
+    @Override
+    public void onLowMemory() {
+    }
+
+    @Override
+    public void onTrimMemory(int i) {
```

Comment:
Naming: can we name the `i` parameter something more meaningful, such as what it's called in the documentation?